scopes/workspace/workspace/workspace.ts

```diff
@@ -1305,6 +1307,25 @@ export class Workspace implements ComponentFactory {
    * @memberof Workspace
    */
   async install(packages?: string[], options?: WorkspaceInstallOptions) {
+    if (options?.addMissingPeers) {
+      const compDirMap = await this.getComponentsDirectory([]);
+      const mergedRootPolicy = this.dependencyResolver.getWorkspacePolicy();
+      const depsFilterFn = await this.generateFilterFnForDepsFromLocalRemote();
+      const pmInstallOptions: PackageManagerInstallOptions = {
+        dedupe: options?.dedupe,
+        copyPeerToRuntimeOnRoot: options?.copyPeerToRuntimeOnRoot ?? true,
+        copyPeerToRuntimeOnComponents: options?.copyPeerToRuntimeOnComponents ?? false,
+        dependencyFilterFn: depsFilterFn,
+        overrides: this.dependencyResolver.config.overrides,
+      };
+      const missingPeers = await this.dependencyResolver.getMissingPeerDependencies(
+        this.path,
+        mergedRootPolicy,
+        compDirMap,
+        pmInstallOptions
+      );
+      packages = Object.entries(missingPeers).map(([peerName, range]) => `${peerName}@${range}`);
```

Comment:
This might cause a bug in case I did something like
`bit install lodash --add-missing-peers` as you will override my lodash. 
Maybe concat them instead?

Reply:
oh, I don't think this will work. I will throw an exception if packages are sent with `--add-missing-peers`

Reply:
Throwing an exception in such case for now.